src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -291,6 +291,15 @@ impl Coordinator {
                     internal,
                     size,
                 } => {
+                    // Only internal users have access to INTERNAL and BILLED AS
+                    if !session.user().is_internal() && (internal || billed_as.is_some()) {
```

Comment:
This allows both `mz_support` and `mz_system`. Is `mz_support` supposed to be allowed access here? If not, change to `is_system_user` and add a test that mz_support is disallowed.

Reply:
We allow `mz_support` already, this check is the same as the one for managed clusters. I'd rather not change it in this PR if we want to change it.